src/links/v1/providers.json

```diff
@@ -63,6 +63,17 @@
         "link_type": "external"
       }
     },
+    {
+      "type": "links",
+      "id": "MatCloud",
```

Comment:
As per [section 3.4](https://github.com/Materials-Consortia/OPTIMADE/blob/v1.0.0/optimade.rst#database-provider-specific-namespace-prefixes) in the specification, database-specific identifiers MUST consist of lowercase alphanumerics ([a-z][0-9]).
```suggestion
      "id": "matcloud",
```

Reply:
We've just updated our Python package to validate this field explicitly, which is why the CI is now failing until this is fixed.